setup.sh

```diff
@@ -489,6 +531,7 @@ else
 	echo "logfile=$var_dir/miniserv.log" >> $cfile
 	echo "errorlog=$var_dir/miniserv.error" >> $cfile
 	echo "pidfile=$var_dir/miniserv.pid" >> $cfile
+	echo "bootscript=$bootscript" >> $cfile
```

Comment:
Does this need to be stored in miniserv.conf and bootscript-name ?

Reply:
The variable stored in`miniserv.conf` is not something we currently use. I thought just to store it, just in case.

Reply:
Don't bother, we can read it from that other file if needed.

Reply:
Okay, I'm fine with removing it!